packages/datatrak-web/src/views/LoginPage.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { LoginForm, EMAIL_VERIFICATION_STATUS } from '@tupaia/ui-components';
+import { useLogin } from '../api/mutations';
+import { ROUTES } from '../constants';
+
+const Wrapper = styled.div`
```

Comment:
This is pretty minor but is an easy change. The Box with the LoginForm is below center because this centering doesn't account for the header. It doesn't have to be pixel perfect but it does look a bit low to me. I guess the same will apply for the Register Modal and possibly others so it might make sense to make a change in ui-components?!

Reply:
Would you mind if I changed as part of the register form, since I made it a reusable wrapper there?